### ISSUE TITLE
Getting User Email

### ISSUE THREAD

**Issue by copperstick6** [actions-on-google/actions-on-google-nodejs#50]
Hi!
First off, great job with this package! I really like the simplicity of this package and it really has amazed me/continues to amaze me with regards to what it can do.
That said, I've frequently found it useful to get user email. Is there a way to do so without Account Linking? It doesn't make too much sense that we get to see the user's location (given access by the user), but that we aren't able to access the user's email in the same way.
Thanks.

TLDR: It'd be great if there was a way to get emails similar to the way we currently get the user's location.

**Comment by Canain** [comment]
Hi, you can do this now with [Google Sign In for the Assistant](https://developers.google.com/actions/identity/google-sign-in).

Check the [reference docs](https://actions-on-google.github.io/actions-on-google-nodejs/classes/conversation.user.html#email) for how you can use it in the library.

```javascript
// Dialogflow
const app = dialogflow({
  clientId: CLIENT_ID,
})

app.intent('Default Welcome Intent', conv => {
  conv.ask(new SignIn('To get your account details'))
})

// Create a Dialogflow intent with the `actions_intent_SIGN_IN` event
app.intent('Get Signin', (conv, params, signin) => {
  if (signin.status === 'OK') {
    const email = conv.user.email
    conv.ask(`I got your email as ${email}. What do you want to do next?`)
  } else {
    conv.ask(`I won't be able to save your data, but what do you want to next?`)
  }
})
```
